.github/workflows/backport.yml

```diff
@@ -0,0 +1,17 @@
+name: Backport PR to branch
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # once a day at 13:00 UTC to cleanup old runs
+    - cron: '0 13 * * *'
+
+permissions:
+  contents: write
+  issues: write
+  pull-requests: write
+  actions: write
+
+jobs:
+  backport:
+    uses: dotnet/arcade/.github/workflows/backport-base.yml@main
```

Comment:
The SDK version of this file includes a description for the created PR. What's the default if we don't specify one?
https://github.com/dotnet/sdk/blob/main/.github/workflows/backport.yml

Reply:
Here is an example of this exact workflow in action in one of the source-build repos - https://github.com/dotnet/source-build-reference-packages/pull/792

```
Backport of #OriginalPRLink to {TargetBranch]

/cc @OriginalAuthor
```

Reply:
The default appears identical to what the SDK has specified.